test/integration/scheduler/scoring/priorities_test.go

```diff
@@ -728,6 +729,7 @@ func TestPodAffinityScoring(t *testing.T) {
 
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
+			featuregatetesting.SetFeatureGateEmulationVersionDuringTest(t, utilfeature.DefaultFeatureGate, version.MustParse("1.32"))
```

Comment:
should we only emulate 1.32 for tests which are trying to turn off the feature? same comment applies to other tests calling SetFeatureGateEmulationVersionDuringTest